types/qbittorrent-api-v2/index.d.ts

```diff
@@ -0,0 +1,463 @@
+    eta: number;
+    f_l_piece_prio: boolean;
+    force_start: boolean;
+    hash: string;
+    last_activity: number;
+    magnet_uri: string;
+    max_ratio: number;
+    max_seeding_time: number;
+    name: string;
+    num_complete: number;
+    num_incomplete: number;
+    num_leechs: number;
+    num_seeds: number;
+    priority: number;
+    progress: number;
+    ratio: number;
+    ratio_limit: number;
+    save_path: string;
+    seeding_time_limit: number;
+    seen_complete: number;
+    seq_dl: boolean;
+    size: number;
+    state: string;
+    super_seeding: boolean;
+    tags: string;
+    time_active: number;
+    total_size: number;
+    tracker: string;
+    up_limit: number;
+    uploaded: number;
+    uploaded_session: number;
+    upspeed: number;
+}
+
+export interface TorrentInfo {
+    save_path: string;
+    creation_date: number;
+    piece_size: number;
+    comment: string;
+    total_wasted: number;
+    total_uploaded: number;
+    total_uploaded_session: number;
+    total_downloaded: number;
+    total_downloaded_session: number;
+    up_limit: number;
+    dl_limit: number;
+    time_elapsed: number;
+    seeding_time: number;
+    nb_connections: number;
+    nb_connections_limit: number;
+    share_ratio: number;
+    addition_date: number;
+    completion_date: number;
+    created_by: string;
+    dl_speed_avg: number;
+    dl_speed: number;
+    eta: number;
+    last_seen: number;
+    peers: number;
+    peers_total: number;
+    pieces_have: number;
+    pieces_num: number;
+    reannounce: number;
+    seeds: number;
+    seeds_total: number;
+    total_size: number;
+    up_speed_avg: number;
+    up_speed: number;
+}
+
+export interface Tracker {
+    url: string;
+    status: number;
+    tier: number;
+    num_peers: number;
+    num_seeds: number;
+    num_leeches: number;
+    num_downloaded: number;
+    msg: string;
+}
+
+export interface Webseed {
+    url: string;
+}
+
+export interface Content {
+    name: string;
+    size: number;
+    progress: number;
+    priority: 0 | 1 | 6 | 7;
+    is_seed: boolean;
+    piece_range: number;
+    availability: number;
+}
+
+export interface SearchJob {
+    id: number;
+}
+
+export interface SearchStatus {
+    id: number;
+    status: string;
+    total: number;
+}
+
+export interface SearchResult {
+    descrLink: string;
+    fileName: string;
+    fileSize: number;
+    fileUrl: string;
+    nbLeechers: number;
+    nbSeeders: number;
+    siteUrl: string;
+}
+
+export interface SearchResults {
+    results: SearchResult[];
+    status: string;
+    total: number;
+}
+
+export interface SearchPlugin  {
+    enabled: boolean;
+    fullName: string;
+    name: string;
+    supportedCategories: string[];
+    url: string;
+    version: string;
+}
+
+export type filterString = 'all' | 'downloading' | 'completed' | 'paused' | 'active' | 'inactive' | 'resumed';
+
+export type Categories = object;
+
+export type PeerData = any;
```

Comment:
is this the best description? or is this usualy an object like `Record<string, unknown>`, `Record<string, any>`, { [key: string]: any;}`?

Reply:
Im currently testing it, because It's not documentet and I don't know yet if the perrdata is an object or binary data

Reply:
Ok, my only dent on this `any`, is it can be anything, which is not very helpful anyway. We can skip this for improvements, if proves time cnsuming,